DEVELOPMENT.md

```diff
@@ -0,0 +1,27 @@
+# Developing inline-plz
+
+`inline-plz` is intended to be compatible with python 3.5 or newer, 3.6 is the recommended version for development.
+
+To get started developing inline-plz, we recommend setting up a `pyenv` `virtualenv` and then running `pip` install the development requirements:
```

Comment:
pyenv isn't supported on windows, unfortunately.

Reply:
also i suggest using `venv` because it's built into python 3.x

Reply:
might be best to just link to https://docs.python-guide.org/dev/env/ and let people figure it out from there since there are a million ways to do it. thoughts?